plugins/mongodb-adapter/src/main/java/org/polypheny/db/adapter/mongodb/MongoEnumerator.java

```diff
@@ -196,6 +197,7 @@ private static PolyValue convert( BsonValue o, MongoTupleType type ) {
                     throw new NotImplementedException();
                 }
             }
+            case INTERVAL -> new PolyInterval( o.asDocument().get( "ms" ).asNumber().longValue(), o.asDocument().get( "m" ).asNumber().longValue() );
```

Comment:
Wouldn't it be better to have "ms" and "m" as constants for MongoDB, e.g., INTERVAL_MILLISECONDS_KEY, and INTERVAL_MONTH_KEY or something like this? Should then of course also be used where this is inserted. Or do I misinterpret this?

Reply:
Adjusted